umbraco-cloud/set-up/legacy-umbraco-visual-studio-setup.md

```diff
@@ -0,0 +1,212 @@
+
+This will make the models of your Document Types available with IntelliSense in Visual Studio.
+[You can read more about configuring ModelsBuilder here.](/11/umbraco-cms/reference/templating/modelsbuilder/)
+
+{% hint style="info" %}
+
+If you are using the [Visual Studio Extension for ModelsBuilder](https://marketplace.visualstudio.com/items?itemName=ZpqrtBnk.UmbracoModelsBuilderExtension&ssr=false) and get the error message "Unauthorized" when generating models, you will need to use or create a backoffice user in your local installation, and supply the credentials for this user in the Visual Studio options. This is necessary because the extension is not able to authenticate against Umbraco Id.
+
+{% endhint %}
+
+### Using Umbraco namespaces in your `*.Core` project
+In order to use Umbraco's features in your `*.Core` project, you have to add references to the DLLs in your `*.Web/bin`.
+
+You can do this by right-clicking on **References** and selecting **Add Reference**. Browse and select the DLLs you'd like to use and then hit **OK**. Don't forget to build.
+
+![References](images/references.gif)
+
+## Git - what should be committed
+
+When working with this solution setup it's important to remember that you have one git repository for your source code, and one within the `*.Web` folder for committing and deploying your changes to Umbraco Cloud.
+
+The cloned git repository from Umbraco Cloud comes with its own `.gitignore` so files that should NOT be committed are already handled.
+
+As a rule of thumb all files that are required to run the Umbraco site should be committed to the git repository in the `*.Web` folder and deployed to Umbraco Cloud. This includes assemblies (`*.dll`).
+
+{% hint style="info" %}
+
+To ensure that your `.dll` files are created in release mode, ensure that you switch to "Release" (instead of "Debug") mode when building the project.
+
+It is recommend to build the project in release mode, before deploying the changes through Git.
+
+{% endhint %}
+
+For the `*.Core` part of the solution as well as the solution file and default `.gitignore` file you commit that to the source code repository. You should ideally set a remote for this git repository to your own git host like GitHub, BitBucket or Visual Studio Team Services.
+
+These are the files and folders you typically want to commit in your own source code repository:
+
+* The project and code files in `*.Core`
+* The solution file `*.sln`
+* `.gitignore`
+* UaaSClone.cmd (used for re-establishing the `*.Web` folder with the git repository from Umbraco Cloud)
+
+## Setup for new team members
+
+When you are working in a team you will have additional people that will use this same setup, but they will only clone your source code repository from your GitHub, Bitbucket or Visual Studio Team Services account. In doing so they will, by default, not get the `*.Web` folder and the Umbraco site, because that part is not contained within the source code repository.
```

Comment:
```suggestion
When you are working in a team you will have additional people that will use this same setup. However, they will only clone your source code repository from your GitHub, Bitbucket, or Visual Studio Team Services account. They will, by default, not get the `*.Web` folder and the Umbraco site, because that part is not contained within the source code repository.
```